docroot/modules/custom/uiowa_hours/src/Form/HoursFilterForm.php

```diff
@@ -86,18 +90,35 @@ public function buildForm(array $form, FormStateInterface $form_state, $config =
       ];
     }
 
-    $result = $this->hours->getHours($config['resource']);
     $form_id = $form_state->getBuildInfo()['form_id'];
     $result_id = $form_id . '_result';
+
     $form['results'] = [
       '#type' => 'container',
       '#attributes' => [
         'role' => 'region',
         'aria-live' => 'assertive',
+        'id' => $result_id,
+        'class' => [
+          'uiowa-hours-container',
+        ],
+      ],
+    ];
+
+    $form['submit'] = [
+      '#type' => 'submit',
+      '#value' => $this->t('Submit'),
+      '#ajax' => [
+        'callback' => '::dateFilterCallback',
+        'wrapper' => 'results-container',
+        'disable-refocus' => TRUE,
+      ],
+      '#attributes' => [
+        'class' => [
+          'element-invisible',
```

Comment:
@joewhitsitt is the "right" thing to use here?

Reply:
if the submit button is only there for the js to use, then it should probably be a display: none instead but i don't think we have a utility class for that available and i wonder if the js would still be able to act on it.

Reply:
Did a quick non-JS test and things aren't working so I might just go with this for now. Thanks.